src/api/app/controllers/webui/project_controller.rb

```diff
@@ -901,8 +901,9 @@ def set_maintained_project
 
   def load_project_info
     find_maintenance_infos
-    @ipackages = @project.linked_packages.map { |package| [ package.name, package.project.name ] }.sort
-    @packages = @project.packages.order(name: :desc).pluck(:name)
+    @ipackages = @project.linked_packages.map { |package| [ package.name, package.project.name ] }
```

Comment:
It seems so. At least for that project. I would risk it and see if this is also true in OBS online